src/luacheck/main.lua

```diff
@@ -2,7 +2,7 @@ local argparse = require "luacheck.argparse"
 local builtin_standards = require "luacheck.builtin_standards"
 local config = require "luacheck.config"
 local fs = require "luacheck.fs"
-local luacheck = require "luacheck"
+local luacheck = require "luacheck.init"
```

Comment:
Are not that equivalent (here and below)?

Reply:
It's not. There is also luacheck.lua. In build time we change project structure and an error occurs.

Reply:
Can we discuss why it didn’t occur for building againts lua?

Reply:
Didn't understand you. I'd like to discuss verbally.

Reply:
Discussed verbally. Decided to use `require "luacheck" ` and therefore keep original requires. Also decided to remove /bin/luacheck.lua and call src/luacheck/main.lua from luacheck.sh instead.

Reply:
Yep. main.lua shoudl contain shebang and has executable bit.

Reply:
Discussed again. Nope. It is better to have separate luacheck executable with #!/usr/bin/env tarantool shebang, which just requires luacheck.main.